### ISSUE TITLE
Last line of file is not visible in home page

### ISSUE THREAD

**Issue by bradyt** [bradyt/cone#20]
A user reported this via email, that they cannot scroll to the last line of the file. I had noticed this occasionally, so can confirm.

**Comment by bradyt** [comment]
Quick note to self that SafeArea might fix this. Though I think user said this doesn't occur for recent update.

**Comment by Aquan1412** [comment]
I'm also seeing this issue with my ledger file. However, it's not just the last line, but rather something like the last 20-25 lines (the whole file has approximately 3000 lines) that are not visible.

**Comment by bradyt** [comment]
Aha, thank you for pointing it out. I can reproduce.

I can only see line 7263, out of 7342 lines, in the test file, https://github.com/simonmichael/hledger/blob/master/examples/bcexample.hledger.

In other words, when I scroll to bottom, last line I see is,

```
; 2014-08-29 price VEA                                     120.04 USD
```

**Comment by bradyt** [comment]
I believe I have a fix, just have to clean my code up a little. I think there was too much text to just be putting into a `SingleChildScrollView`. I tried splitting the text on new lines and using a `ListView`, and it seems to resolve the issue. Hope to have it available soon.

**Comment by bradyt** [comment]
This will be fixed in v0.2.5+13. Thank you again for the reports.
